### PR TITLE
Cb/psi test updates

### PR DESCRIPTION
**Pull request by claytonpbarrows**

### CODE REVIEW

**Comment by claytonpbarrows** [comment]
all the tests run. there are still 48 failures

File: test/test_operations_solve.jl [hunk 1]
```diff
@@ -397,10 +397,7 @@ end
     parameters_value = [true, false]
     systems = [c_sys5, c_sys5_dc]
     networks = [DCPPowerModel, NFAPowerModel, StandardPTDFModel, CopperPlatePowerModel]
-    PTDF_ref = IdDict{System, PTDF}(
-        c_sys5 => build_PTDF5(),
-        c_sys5_dc => build_PTDF5_dc(),
-    )
+    PTDF_ref = IdDict{System, PTDF}(c_sys5 => build_PTDF5(), c_sys5_dc => build_PTDF5_dc())
```
Comment (jd-lara) [inline test/test_operations_solve.jl:400]: @daniel-thom made a fix in IS to avoid the issue with the UUID

**Review by jd-lara — APPROVED** [review]
LGTM. Thanks for the fixes. @sourabhdalvi and I discussed a game plan to address the remaining failures.

File: test/test_operations_solve.jl (hunk 1 above)
Comment (claytonpbarrows) [reply test/test_operations_solve.jl:400]: good. I don't think it really matters here, so I'll leave it as is unless anyone cares.

**Pull request merged**